editor/project_converter_3_to_4.cpp

```diff
@@ -1008,6 +1008,7 @@ static const char *gdscript_properties_renames[][2] = {
 	//	{ "cast_to", "target_position" }, // RayCast2D, RayCast3D
 	//	{ "doubleclick", "double_click" }, // InputEventMouseButton
 	//	{ "group", "button_group" }, // BaseButton
+	//  { "percent_visible, "show_percentage}, // ProgressBar, conflicts with Label and RichTextLabel, but may be a worth it.
```

Comment:
Indentation issue but I wouldn't include it if it's not meant to be used.
This list says it's about words which are too common to be blindly replaced, which is not the case for those.

Reply:
All good now